TEMPLATES.md

```diff
@@ -111,3 +111,10 @@ before gathering the artifacts.
 `ci-operator.openshift.io/always-show-output`:
 Will output the logs of all the containers in the pod, no matter what the exit code was.
 The value should be `true` to enable this feature.
+
+`ci-operator.openshift.io/always-show-container-output`:
+Will output the logs of the specified containers after the pod fails, if
+they are in a terminated state.
+
+**Note:**
+To specify multiple containers in `ci-operator.openshift.io/wait-for-container-artifacts` and `ci-operator.openshift.io/always-show-container-output` annotations, the value should be splited by `,`.
```

Comment:
Perhaps include this into the descriptions themselves? This would require rewritting all of them into a different form, but I think it would be worth the improvement.

_Comma-separated list of container names for which ci-operator will wait until complete / for which ci-operator will collect and log their output if pod fails_